test/Sentry.Unity.Tests/IntegrationTests.cs

```diff
@@ -274,6 +275,8 @@ public IEnumerator DebugLogException_OnMainThread_IsCapturedAndIsMainThreadIsTru
     [UnityTest]
     public IEnumerator DebugLogException_InTask_IsCapturedAndIsMainThreadIsFalse()
     {
+        LogAssert.Expect(LogType.Exception, new Regex("Exception: .* Test Event"));
```

Comment:
Fixed a flaky test while at it.

Reply:
Not sure this fixed it: 
![image](https://github.com/user-attachments/assets/3fc48838-f018-4a56-b11d-d60f424d5587)
https://github.com/getsentry/sentry-unity/actions/runs/11224732057/job/31202068887